Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2156,7 +2156,15 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin()
     if (videoSinkPad)
         g_signal_connect_swapped(videoSinkPad.get(), "notify::caps", G_CALLBACK(videoSinkCapsChangedCallback), this);
 #endif
+#if USE(WESTEROS_SINK)
```

Comment:
We use WESTEROS_SINK when HOLE_PUNCH_GSTREAMER=ON and createVideoSink is called whtn HOLE_PUNCH_GSTREAMER=OFF. So moving it to base may not work.